### ISSUE TITLE
adding media such as an ISO to a bhyve vm.

### ISSUE THREAD

**Issue by matthewearl00** [cbsd/cbsd#141]
I needed to add an iso to a windows install because I needed to install the virtio drivers on the windows guest. I could not find how to do this without manually running a bhyve string (which is what i did). When I would change the bhyve.conf in the jails-system directory it would change back as soon as I ran the vm.  I saw a media command and tried running it but could not get past the path= part.

ex. cbsd media mode=attach name=virtio.iso jname=EXAMPLEVM path=/usr/jails/src/iso/
would result in jname - doesnt have that path or something.

I would like to know the correct syntax or if this is a bug.  THanks

**Comment by olevole** [comment]
**cbsd media** command still unfinished ;-) so it is the reason why it is still undocumented.

As work-around, you can put ISO in that name which CBSD expects ( for Windows 10 profile ):

~~~
ln -sf <path-to-ISO> /usr/jails/src/iso/cbsd-iso-windows10.iso
~~~

where <path-to-ISO> - is your real custom ISO image.

**Comment by Dorlas** [comment]
Another working durty hack:
1) Edit bhyve.conf (for example): /CBSD/jails-system/windows10/bhyve.conf
2) On parameter: cd_args='' place your ISO: (exampe: cd_args=' -s 7:0,ahci-cd,/CBSD/src/iso/virtio.iso')
3) Start VM without CBSD: sh /CBSD/share/bhyverun.sh -c /CBSD/jails-system/windows10/bhyve.conf

After connect to VM your can see new Virtual CD/DVD with your ISO file image.

Copy file, install and shutdown. Next run VM with cbsd bstart.

**Comment by olevole** [comment]
Implemented: https://github.com/cbsd/cbsd/commit/b1bd59770e9f2b7779f9f95fd8ee073ab400d9bc
Correct behavior will be available in next release ( 11.0.12 )
